po/ja.po

```diff
@@ -282,7 +284,7 @@ msgstr "1 分"
 #: pkg/docker/containers-view.jsx:614
 msgid "1 Vulnerability"
 msgid_plural "$0 Vulnerabilities"
-msgstr[0] ""
+msgstr[0] "1 脆弱性"
```

Comment:
This also does not seem ideal. (note that jp does not have plurals). Not really sure what would happen if there would be some different number of vulnerabilities. (but this is not why tests are failing).

Reply:
https://github.com/cockpit-project/cockpit/pull/13407 fixes this